### ISSUE TITLE
ITC-3 A0302 cannot be acknowledged or suspended on core 3.1.2 or 3.1.3

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_validator#446]
These two test fail on core 3.1.2 and 3.1.3:

```
rspec ./spec/site/tlc/alarm_spec.rb:58 # Site::Traffic Light Controller Alarm A0302 can be acknowledged
rspec ./spec/site/tlc/alarm_spec.rb:109 # Site::Traffic Light Controller Alarm A0302 can be suspended and resumed
```

The tests acknowledges/suspends an Alarm, then waits for an updated Alarm to be received from the device, but it never arrives.

**Comment by sveitech** [comment]
I think you need to update the schema. This file: https://github.com/rsmp-nordic/rsmp_schema/blob/main/schemas/core/3.1.2/alarm_suspended_resumed.json

needs to have the "inactive" option added as well.

**Comment by emiltin** [comment]
"inactive" has been added to the schema.

**Comment by emiltin** [comment]
But currently the ITC-3 seems not to connect: Site did not connect within 120s

**Comment by emiltin** [comment]
Fixed in #434
